azure-pipelines.yml

```diff
@@ -67,8 +68,12 @@ stages:
           commonMavenArguments: -B -Pdeploy-sonarsource -Dmaven.test.skip=true
         steps:
           - checkout: self
+
           - task: NuGetToolInstaller@1
             displayName: "Install NuGet"
+            inputs:
+              versionSpec: $(NUGET_VERSION)
```

Comment:
Why do we need to specify the version now?

Reply:
It was specified for [another Task for the ITs](https://github.com/SonarSource/sonar-scanner-msbuild/blob/5e3f8d134003ddef09f9b0d8811fa84902eeeddd/azure-pipelines.yml#L455), so I thought let's use the same version for both.
Why not leave it empty? So that if a new version is released then it won't fail the pipeline if it contains a breaking change or bug.
WDYT?

Reply:
Good thinking!